2025-03-29 17:32:33,439 INFO    MainThread:45881 [wandb_setup.py:_flush():67] Current SDK version is 0.19.8
2025-03-29 17:32:33,439 INFO    MainThread:45881 [wandb_setup.py:_flush():67] Configure stats pid to 45881
2025-03-29 17:32:33,439 INFO    MainThread:45881 [wandb_setup.py:_flush():67] Loading settings from /Users/liuyihao/.config/wandb/settings
2025-03-29 17:32:33,439 INFO    MainThread:45881 [wandb_setup.py:_flush():67] Loading settings from /Users/liuyihao/Desktop/m2cw/wandb/settings
2025-03-29 17:32:33,439 INFO    MainThread:45881 [wandb_setup.py:_flush():67] Loading settings from environment variables
2025-03-29 17:32:33,440 INFO    MainThread:45881 [wandb_init.py:setup_run_log_directory():647] Logging user logs to /Users/liuyihao/Desktop/m2cw/wandb/run-20250329_173233-uqfa331w/logs/debug.log
2025-03-29 17:32:33,440 INFO    MainThread:45881 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to /Users/liuyihao/Desktop/m2cw/wandb/run-20250329_173233-uqfa331w/logs/debug-internal.log
2025-03-29 17:32:33,440 INFO    MainThread:45881 [wandb_init.py:monkeypatch_ipython():599] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x33f43fbc0>
2025-03-29 17:32:33,440 INFO    MainThread:45881 [wandb_init.py:init():761] calling init triggers
2025-03-29 17:32:33,440 INFO    MainThread:45881 [wandb_init.py:init():766] wandb.init called with sweep_config: {}
config: {'_wandb': {}}
2025-03-29 17:32:33,440 INFO    MainThread:45881 [wandb_init.py:init():784] starting backend
2025-03-29 17:32:33,440 INFO    MainThread:45881 [wandb_init.py:init():788] sending inform_init request
2025-03-29 17:32:33,484 INFO    MainThread:45881 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2025-03-29 17:32:33,484 INFO    MainThread:45881 [wandb_init.py:init():798] backend started and connected
2025-03-29 17:32:33,486 INFO    MainThread:45881 [wandb_run.py:_label_probe_notebook():1204] probe notebook
2025-03-29 17:32:33,487 INFO    MainThread:45881 [wandb_init.py:init():891] updated telemetry
2025-03-29 17:32:33,515 INFO    MainThread:45881 [wandb_init.py:init():915] communicating run to backend with 90.0 second timeout
2025-03-29 17:32:33,848 INFO    MainThread:45881 [wandb_init.py:init():990] starting run threads in backend
2025-03-29 17:32:33,899 INFO    MainThread:45881 [wandb_run.py:_console_start():2375] atexit reg
2025-03-29 17:32:33,899 INFO    MainThread:45881 [wandb_run.py:_redirect():2227] redirect: wrap_raw
2025-03-29 17:32:33,899 INFO    MainThread:45881 [wandb_run.py:_redirect():2292] Wrapping output streams.
2025-03-29 17:32:33,899 INFO    MainThread:45881 [wandb_run.py:_redirect():2315] Redirects installed.
2025-03-29 17:32:33,901 INFO    MainThread:45881 [wandb_init.py:init():1032] run started, returning control to user process
2025-03-29 17:32:33,902 INFO    MainThread:45881 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-29 17:32:33,902 INFO    MainThread:45881 [wandb_init.py:_pause_backend():564] pausing backend
2025-03-29 17:32:33,905 INFO    MainThread:45881 [wandb_init.py:_resume_backend():569] resuming backend
2025-03-29 17:33:07,432 INFO    MainThread:45881 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-29 17:33:07,432 INFO    MainThread:45881 [wandb_init.py:_pause_backend():564] pausing backend
2025-03-29 17:33:07,437 INFO    MainThread:45881 [wandb_init.py:_resume_backend():569] resuming backend
2025-03-29 17:33:07,439 INFO    MainThread:45881 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-29 17:33:07,439 INFO    MainThread:45881 [wandb_init.py:_pause_backend():564] pausing backend
2025-03-29 17:33:07,441 INFO    MainThread:45881 [wandb_init.py:_resume_backend():569] resuming backend
2025-03-29 17:33:07,442 INFO    MainThread:45881 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-29 17:33:07,442 INFO    MainThread:45881 [wandb_init.py:_pause_backend():564] pausing backend
2025-03-29 17:33:07,444 INFO    MainThread:45881 [wandb_init.py:_resume_backend():569] resuming backend
2025-03-29 17:33:08,996 INFO    MainThread:45881 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-29 17:33:08,996 INFO    MainThread:45881 [wandb_init.py:_pause_backend():564] pausing backend
2025-03-29 17:33:09,006 INFO    MainThread:45881 [wandb_init.py:_resume_backend():569] resuming backend
2025-03-29 17:33:09,897 INFO    MainThread:45881 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-29 17:33:09,898 INFO    MainThread:45881 [wandb_init.py:_pause_backend():564] pausing backend
2025-03-29 17:33:09,902 INFO    MainThread:45881 [wandb_init.py:_resume_backend():569] resuming backend
2025-03-29 17:33:11,072 INFO    MainThread:45881 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-29 17:33:11,072 INFO    MainThread:45881 [wandb_init.py:_pause_backend():564] pausing backend
2025-03-29 17:33:11,083 INFO    MainThread:45881 [wandb_init.py:_resume_backend():569] resuming backend
2025-03-29 17:33:11,088 INFO    MainThread:45881 [wandb_init.py:setup_run_log_directory():647] Logging user logs to /Users/liuyihao/Desktop/m2cw/wandb/run-20250329_173311-4avig8c1/logs/debug.log
2025-03-29 17:33:11,088 INFO    MainThread:45881 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to /Users/liuyihao/Desktop/m2cw/wandb/run-20250329_173311-4avig8c1/logs/debug-internal.log
2025-03-29 17:33:11,088 INFO    MainThread:45881 [wandb_init.py:init():761] calling init triggers
2025-03-29 17:33:11,088 INFO    MainThread:45881 [wandb_init.py:init():766] wandb.init called with sweep_config: {}
config: {'_wandb': {}}
2025-03-29 17:33:11,088 INFO    MainThread:45881 [wandb_init.py:init():773] finishing previous run: uqfa331w
2025-03-29 17:33:11,089 INFO    MainThread:45881 [wandb_run.py:_finish():2112] finishing run liuyihao649-university-of-cambridge/Qwen_baseline/uqfa331w
2025-03-29 17:33:11,089 ERROR   MainThread:45881 [jupyter.py:save_history():450] Run pip install nbformat to save notebook history
2025-03-29 17:33:11,089 INFO    MainThread:45881 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-29 17:33:11,089 INFO    MainThread:45881 [wandb_init.py:_jupyter_teardown():581] cleaning up jupyter logic
2025-03-29 17:33:11,089 INFO    MainThread:45881 [wandb_run.py:_atexit_cleanup():2340] got exitcode: 0
2025-03-29 17:33:11,089 INFO    MainThread:45881 [wandb_run.py:_restore():2322] restore
2025-03-29 17:33:11,089 INFO    MainThread:45881 [wandb_run.py:_restore():2328] restore done
2025-03-29 17:33:11,826 INFO    MainThread:45881 [wandb_run.py:_footer_history_summary_info():3956] rendering history
2025-03-29 17:33:11,826 INFO    MainThread:45881 [wandb_run.py:_footer_history_summary_info():3988] rendering summary
2025-03-29 17:33:11,827 INFO    MainThread:45881 [wandb_run.py:_footer_sync_info():3917] logging synced files
